quickstart/101-analysis-services-create/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "resource_group_location" {
+  default     = "eastus"
+  description = "Location for all resources."
+}
+
+variable "resource_group_name_prefix" {
+  default     = "rg"
+  description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
+}
+
+variable "server_name" {
+  description = "The name of the Azure Analysis Services server to create. Server name must begin with a letter, be lowercase alphanumeric, and between 3 and 63 characters in length. Server name must be unique per region."
```

Comment:
Our e2e test cannot work with `variable` without default value, could you please add a default value for this? Thanks!

Reply:
Fixed